### ISSUE TITLE
I think a New Folder button is warranted next to Up and Refresh/Reload buttons

### ISSUE THREAD

**Issue by carlosvsilva** [files-community/Files#3228]
**Is your feature request related to a problem? Please describe.**
Feature request for a "New Folder" button in the top row of icons and while there, maybe add a Properties (with a 'info' icon) after.

**Describe the solution you'd like**
The Files team could add a New Folder and Properties buttons to the top button row, maybe configurable as on/off in the options.

**Describe alternatives you've considered**
I can only create a new folder through a right click and searching for New-> New Folder and going from there.

**Additional context**
![Files New Folder](https://user-images.githubusercontent.com/40889421/105521061-58a51200-5cd3-11eb-8cb1-b64bd4aaf20d.png)

**Comment by yaichenbaum** [comment]
@carlosvsilva There is already an option to create new items in the three dot menu, can you check if that works for you?

**Comment by carlosvsilva** [comment]
Right I just noticed it, had been there before but didn't imediately associate the "New" with creating a directory, but it makes sense.

I just think it's a central file manager function to create new folders (and new files), and having to go through 3 different steps to create the folder seems a tad excessive. I would welcome a direct button at the top row to click it directly like I used to do on MacOS X.

Thanks for consideration.
